recipes-core/images/iot2050-image-example.bb

```diff
@@ -81,16 +81,25 @@ IOT2050_DEBIAN_BT_PACKAGES = " \
     bluez \
     pulseaudio-module-bluetooth \
     "
+
 # alsa support
 IOT2050_DEBIAN_ALSA_PACKAGES = " \
     alsa-utils \
     alsa-tools \
     "
+
+# multiarch support
+IOT2050_DEBIAN_MULTIARCH_PACKAGES = " \
+    libc6:armhf \
+    libstdc++6:armhf \
```

Comment:
This should only enable the very basic dependencies in the SDK and on the target. Which cases do we want to address with it?

Reply:
for pn driver,currently it only supports 32bit.user could port the pn to iot2050 and also could create their pn application by themselves.